pkg/compiler/syscall.go

```diff
@@ -23,6 +23,7 @@ var syscalls = map[string]map[string]string{
 		"GetTransactionHeight":    interopnames.SystemBlockchainGetTransactionHeight,
 	},
 	"contract": {
+		"Call":                  interopnames.SystemContractCall,
```

Comment:
@alexvanin will be happy to see another breaking change, of course.

Reply:
Though it makes sense probably. It's the only thing in `engine` and it was there since `APPCALL` opcode which is long gone now.